### ISSUE TITLE
debug hover string variable ...more appears

### ISSUE THREAD

**Issue by kzhui125** [Microsoft/vscode-go#1777]
```go
	s1 := "你好"
	s2 := "abc"
	fmt.Println(s1)
	fmt.Println(s2)
```
![image](https://user-images.githubusercontent.com/11925432/42421264-57855dda-8305-11e8-9b66-ef5019392920.png)

hover on s1, I get message "你好...+4 more"

“你好” is expected.

**Comment by ramya-rao-a** [comment]
That `more` is coming from https://github.com/Microsoft/vscode-go/blob/0.6.84/src/debugAdapter/goDebug.ts#L823

For s1, delve returns a string with length 6. The string itself is of length 2 and so the +4.

If anyone wants to take a look at how to fix this, then PRs are most welcome.
Wiki for [Running the Go extension from source](https://github.com/Microsoft/vscode-go/wiki/Building,-Debugging-and-Sideloading-the-extension-in-Visual-Studio-Code) and [debugging the debug adapter](https://github.com/Microsoft/vscode-go/tree/master/src/debugAdapter)

**Comment by xmattstrongx** [comment]
Hi Ramya. I am looking for something to help the vscode-go extension and I came across this issue. I have the debugging the debug adapter running and see the code in goDebug.ts hitting as you noted. 

I am still trying to figure out why delve returns a string with length 6 even though the string is only length 2. Id be happy to submit a PR for this issue just a little confused on what the expected outcome of this issue is.

**Comment by kzhui125** [comment]
@xmattstrongx 
```go
len("你好") is 6
len([]rune("你好")) is 2
```

**Comment by ramya-rao-a** [comment]
Looks like 6 is for the number of bytes and 2 is the number of characters.
Thanks for that link @kzhui125!

**Comment by ramya-rao-a** [comment]
This fix is now out in the latest update (0.6.85) to the Go extension. Thanks @xmattstrongx!
